development/scripts/clone-repos.sh

```diff
@@ -5,14 +5,15 @@ user=ENTER_USERNAME
 
 repos=(
 "osc-core" 
-"security-mgr-smc-plugin"
+#"security-mgr-smc-plugin"

```

Comment:
hopefully the plugins should come back under the opensecuritycontroller community soon? :)